src/ensembl/tests/fixtures/entity-viewer/gene.ts

```diff
@@ -0,0 +1,39 @@
+import faker from 'faker';
+import times from 'lodash/times';
+import { scaleLinear } from 'd3';
+
+import { createSlice } from './slice';
+import { createTranscript } from './transcript';
+
+import { Gene } from 'src/content/app/entity-viewer/types/gene';
+import { TicksAndScale } from 'src/content/app/entity-viewer/gene-view/components/base-pairs-ruler/BasePairsRuler';
+
+export const createGene = (): Gene => {
+  const geneSlice = createSlice();
+  const transcript = createTranscript();
+
+  return {
+    type: 'Gene',
+    id: faker.random.uuid(),
+    symbol: faker.lorem.word(),
+    so_term: faker.lorem.word(),
+    slice: geneSlice,
+    transcripts: [transcript]
+  };
+};
+
+const getScale = () => {
+  const domain = [1, faker.random.number()];
+  const range = [0, faker.random.number()];
```

Comment:
probably safer to give these numbers at least slightly meaningful values. For example, if domain starts at 1, faker.random.number() should never be 0; and also it should probably be between a 1000 and 100000. The range's max number shouldn't be outrageously huge — let's say in between 100 and 600.

Reply:
✔️